recipes/xeus-python/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "xeus-python" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "cf335e1e5d91de4d13f0800163b98ac6b52a06302ac304ee772a0429e6ad7b8a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xeus-python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+  {% if 'toolchain' in compiler('cxx') %}skip: True{% endif %}  # Only build with modern compilers.
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - cppzmq >=4.3.0,<4.4.0
+    - zeromq >=4.2.5,<4.4.0
+    - xtl >=0.5.2,<0.6
+    - xeus >=0.18.1,<0.19.0
+    - pybind11 >=2.2.4,<3.0
+    - nlohmann_json >=3.3.0,<4.0
+  run:
+    - jedi >=0.13.1,<0.14
+    - pygments >=2.3.1,<3.0.0
+
```

Comment:
Need `python` here.

Reply:
why?

Reply:
Because it needs libpython which is in the python package.

Reply:
ah, python does not have a `run_exports`.

Reply:
Done!